### ISSUE TITLE
If there is ‘ignore’ in my labels, what should I do with it?

### ISSUE THREAD

**Issue by c-dafan** [ultralytics/yolov5#681]

**Comment by Ownmarc** [comment]
Ignore is a label for a class ? If you want to detect them, keep it, it will have its own class. If you do not need to detect what is labelled as « ignore », simply omit the class in your data.yaml names

**Comment by Aktcob** [comment]
If 'ignore' is not a label or a class, it seemed that the target generator should be updated. As I know, right now, yolov5 does not suppose ignore bounding box.,

**Comment by glenn-jocher** [comment]
There is no method to ignore labels at the moment other than simply removing the label from your training data.

**Comment by yuvalgrossman** [comment]
Hi, so are there any updates on this?

**Comment by glenn-jocher** [comment]
@yuvalgrossman what are you looking for exactly?

**Comment by yuvalgrossman** [comment]
If I have a database prepared for YOLO lets say with 10 classes, and I want to train it once with all classes, and once without one (or several) of them, to see if it improves. I thought maybe there should be a way to ignore that class on training (I guess in the dataloader)

**Comment by glenn-jocher** [comment]
@yuvalgrossman nothing like this exists. You may want to submit a PR for your proposed feature addition.

**Comment by glenn-jocher** [comment]
@yuvalgrossman BTW, the closest example of this I can think of if my copyattr function, which has include and exclude lists, optionally supplied. You might be able to carry over this logic to your idea.
https://github.com/ultralytics/yolov5/blob/2e8e02745b76c59b71aab9c9de2bed6ef9721c82/utils/torch_utils.py#L199-L206

**Comment by Dimlight** [comment]
@glenn-jocher  what about this solution as @Ownmarc mentioned: "simply omit the class in your data.yaml names".Or do I need to go through all images to remove the labels one by one?

**Comment by glenn-jocher** [comment]
@Dimlight That's a viable solution as well. It depends on which approach is more convenient for you: manipulating the data.yaml or manually removing the labels from each image.
